### ISSUE TITLE
[I18N] "{time_span} ago" in Transmission GTK

### ISSUE THREAD

**Issue by meskobalazs** [transmission/transmission#3214]
Currently there are 9 string containing this message part. The `{time_span}` part is put together using this function:

https://github.com/transmission/transmission/blob/9eeb3db89c2a09581298ea481fdaf3e5c92e34ec/gtk/Utils.cc#L89

So we really get a time span. This is fine in English, but in other languages the `{time_span}` text should be different, e.g. when talking about the past or the future. Let's see two specific examples in Hungarian (the language I translate to):

| English | Hungarian |
| --- | --- |
|1 minute|1 perc|
|1 minute ago|1 perce|
|in 1 minute|1 perc múlva|
|1 month|1 hónap|
|1 month ago|1 hónapja|
|in 1 month|1 hónap múlva|

So basically when talking about the future, I can make it work quite easily. But the past version are impossible to solve with correct grammar.

I'd propose having two versions of the strings (by adding a new parameter to the function), but that might only solve my use case, and I am quite sure this problem is not limited to Hungarian, so the perspectives of other translators might be useful.

**Comment by ckerr** [comment]
@meskobalazs we don't have a large number of translators who actively check the issue tracker. IMO the best chances for improvement would be if you could make a PR with the changes that would fix your use case.

**Comment by ckerr** [comment]
@meskobalazs how would one say "1 minute remaining"?

**Comment by meskobalazs** [comment]
Thanks! This worked great.
